### DISCUSSION TITLE
my fooocus can't work: error: expecting value

### DISCUSSION THREAD

**Discussion by funkysugar1983** [lllyasviel/Fooocus#3736] [Q&A] [answered]
![微信截图_20241114015212](https://github.com/user-attachments/assets/0d2ec693-2006-43f8-9f4b-246b2ebddfb1)
![微信截图_20241114015234](https://github.com/user-attachments/assets/f4fa0ad3-17b9-4a5b-9966-e2a687a230f2)
last time i used it was about 1 month ago, it was normal.
but today when i ran it, no function works.  it says as the picture i uploaded.
can anyone help me? thx a lot.

**Comment by mashb1t** [comment, marked as answer]
Please follow the [troubleshooting guide](https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md).

**Reply by funkysugar1983** [reply]
thx for reply, but i can't find the problem i'm facing in that guide.....
